README.md

```diff
@@ -23,51 +23,37 @@ There's no need to install anything to use this, it's a self-contained Go projec
 
 ### Define a scenario
 
-Scenarios are defined using plain Go code. They are located in the [scenarios](./scenarios/) folder.
+Scenarios are defined using plain Go code. They are located in the [scenarios](./scenarios/) folder. There are already
```

Comment:
Long term we'll want this OSS repo to not contain any baked in scenarios, right?

Reply:
Not from my POV. This should have all scenarios we want to run as a company. This is a tool not a library IMO; we don't want the compatibility guarantees that come with the latter.

Reply:
Hmm.. when we last spoke I thought we agreed otherwise.
But I'm totally fine starting out like this and breaking it out to be extensible eventually.

Reply:
I probably contradicted myself. But at this point, we really need to shoot for simplicity and necessity and just be willing to evolve.